deploy/olm-catalog/nxiq-operator-certified/1.94.0-1/nxiq-operator-certified.v1.94.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,368 @@
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "sonatype.com/v1alpha1",
+          "kind": "NexusIQ",
+          "metadata": {
+            "name": "example-nexusiq"
+          },
+          "spec": {
+            "deployment": {
+              "postStart": {
+                "command": null
+              },
+              "preStart": {
+                "command": null
+              }
+            },
+            "fullnameOverride": "",
+            "imagePullSecrets": [],
+            "ingress": {
+              "annotations": {},
+              "enabled": false,
+              "hosts": [],
+              "path": "/",
+              "tls": []
+            },
+            "iq": {
+              "adminPort": 8071,
+              "applicationPort": 8070,
+              "configYaml": {
+                "createSampleData": true,
+                "server": {
+                  "adminConnectors": [
+                    {
+                      "port": 8071,
+                      "type": "http"
+                    }
+                  ],
+                  "applicationConnectors": [
+                    {
+                      "port": 8070,
+                      "type": "http"
+                    }
+                  ]
+                },
+                "sonatypeWork": "/sonatype-work"
+              },
+              "imageName": "registry.connect.redhat.com/sonatype/nexus-iq-server:1.94.0-ubi-1",
+              "imagePullPolicy": "IfNotPresent",
+              "imagePullSecret": "",
+              "licenseSecret": "",
+              "memory": "1Gi",
+              "name": "nxiq"
+            },
+            "nameOverride": "",
+            "persistence": {
+              "accessMode": "ReadWriteOnce",
+              "storageSize": "1Gi"
+            },
+            "service": {
+              "annotations": {},
+              "enabled": false,
+              "labels": {},
+              "ports": [
+                {
+                  "name": "nexus-service",
+                  "port": 80,
+                  "targetPort": 80
+                }
+              ]
+            },
+            "serviceAccount": {
+              "create": true,
+              "name": null
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: "Security"
+    description: |-
+      Nexus Lifecycle is an open source governance platform that continuously identifies risk,
+      automatically enforces policy, and provides visibility throughout the entire SDLC.
+    containerImage: registry.connect.redhat.com/sonatype/nxiq-operator-certified:1.94.0-1
+    repository: https://github.com/sonatype/operator-iq
+    createdAt: 2020-05-01
+    support: Sonatype
+    certified: "true"
+  name: nxiq-operator-certified.v1.94.0-1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: NexusIQ
+      name: nexusiqs.sonatype.com
+      version: v1alpha1
```

Comment:
I've got no idea how this relates to the actual version - @jflinchbaugh can you school me how this is used/exposed to end users?

Reply:
Honestly, I don't know how they really matter. I'm just keeping them as they were from generation. When using these operators, when you instantiate it, it'll generate this yaml file for you, so you can customize. This is one of the things you'd never touch. It just needs to match. I think it's just versioning it preemptively like an API in case you ever wish it had been versioned. v1alpha1 is a common default. You'll see it even on these standard operator/kubernetes APIs were using.